src/main/java/org/iac2/service/architecturereconstruction/plugin/implementation/opentoscacontainer/DockerContainerEnhancementPlugin.java

```diff
@@ -0,0 +1,98 @@
+package org.iac2.service.architecturereconstruction.plugin.implementation.opentoscacontainer;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.model.Container;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient;
+import com.github.dockerjava.transport.DockerHttpClient;
+import com.google.common.collect.Maps;
+import io.github.edmm.model.DeploymentModel;
+import io.github.edmm.model.component.RootComponent;
+import io.github.edmm.model.relation.HostedOn;
+import io.github.edmm.model.support.EdmmYamlBuilder;
+import org.apache.commons.compress.utils.Lists;
+import org.eclipse.winery.model.tosca.utils.ModelUtilities;
+import org.iac2.common.model.InstanceModel;
+import org.iac2.common.model.ProductionSystem;
+import org.iac2.service.architecturereconstruction.common.interfaces.ModelEnhancementPlugin;
+import org.iac2.service.utility.Edmm;
+import org.iac2.service.utility.Utils;
+
+import java.time.Duration;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DockerContainerEnhancementPlugin implements ModelEnhancementPlugin {
+    @Override
+    public Collection<String> getRequiredPropertyNames() {
+        Collection<String> reqs = Lists.newArrayList();
+        return reqs;
+    }
+
+    @Override
+    public String getIdentifier() {
+        return "docker-enhancement-plugin";
+    }
+
+    @Override
+    public InstanceModel enhanceModel(InstanceModel instanceModel, ProductionSystem productionSystem) {
```

Comment:
in what way?

you mean that we have two instances of an instance model? so that input and output are to separate objects? Or you mean literally the same?